tools/yaml-templates/security.yml

```diff
@@ -35,9 +35,8 @@ steps:
       IsShipped: false
       DropsToRetain: 'CodeAnalysisLogs'
     condition: and(
-      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
```

Comment:
(Asking because I genuinely don't know) why do we want to remove the succeeded condition?

Reply:
It's a good question.
This condition doesn't really do anything. A condition of succeeded or succeededWithIssues is the default behavior. We should either set this to succeeded only, or to nothing.

From here: [Conditions](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml)
> By default, a step runs if nothing in its job has failed yet and the step immediately preceding it has finished

Reply:
We have a bunch of these across the yml files- what do you think about me creating a ticket under engineering improvement to take care of these in another PR?

Reply:
@aeunms That would be great, thank you!

Reply:
Thanks @aeunms !